apps/gallery/js/ImageEditor.js

```diff
@@ -276,7 +285,12 @@ var exposureSlider = (function() {
     pixel -= thumbWidth / 2;
 
     // Move the thumb to that position
-    thumb.style.left = pixel + 'px';
+    if (navigator.mozL10n.language.direction === 'ltr') {
+      thumb.style.left = pixel + 'px';
+    }
+    else {
+      thumb.style.right = pixel + 'px';
+    }
```

Comment:
If you hard-coded left and right to 0, could you just set style.marginStart here to move the thumb?